src/core/lib/gprpp/thd_posix.cc

```diff
@@ -48,6 +49,24 @@ struct thd_arg {
   bool tracked;
 };
 
+const size_t page_size = static_cast<size_t>(sysconf(_SC_PAGESIZE));
+
+size_t RoundUpToPageSize(size_t size) {
+  return (size + page_size - 1) & ~(page_size - 1);
+}
+
+// Return the minimum valid stack size that can be passed to
```

Comment:
"Returns"

As Google C++ style guide suggests, comments should be descriptive rather than imperative.